src/main.rs

```diff
@@ -1,6 +1,6 @@
-mod pre_compute;
-mod utils;
+mod api;
+mod compute;
 
 fn main() {
-    println!("Hello, world!");
+    compute::app_runner::start();
```

Comment:
The pre-compute process should exit with the return code provided by the `start` method
```suggestion
    process::exit(compute::app_runner::start());
```